src/03-ticket-calculator.js

```diff
@@ -54,7 +54,27 @@ const exampleTicketData = require("../data/tickets");
     calculateTicketPrice(tickets, ticketInfo);
     //> "Entrant type 'kid' cannot be found."
  */
-function calculateTicketPrice(ticketData, ticketInfo) {}
+function calculateTicketPrice(ticketData, ticketInfo) {
+  const { ticketType, entrantType, extras } = ticketInfo;
+
+
+  const admission = ticketData[ticketType];
+  if (typeof admission !== 'object') return `Ticket type 'incorrect-type' cannot be found.`;
```

Comment:
This is considered hardcoded. If the ticket type was "VIP", I should expect to get the error message `Ticket type 'VIP' cannot be found.`;